pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -166,14 +166,6 @@ stdenv.mkDerivation {
     # all our root unit dirs if they are symlinks. This does exactly what we
     # need (AFAICT).
     ./0019-core-handle-lookup-paths-being-symlinks.patch
```

Comment:
Minor nit: We should add a link to https://github.com/systemd/systemd/pull/20479 here, so the PR sent upstream can be found easier.